README.md

```diff
@@ -67,20 +67,6 @@ For more details on the snap, including how to build and install it, please refe
 - The current targeted version of the Go language runtime for release artifacts is v1.18.x
 - The minimum supported version of the Go language runtime is v1.18.x
 
-##### ZeroMQ
```

Comment:
should we mention ZMQ is no longer supported in document after 3.0 + instead (due to historical reason) ?